### ISSUE TITLE
Mangos Bot addon compatibility

### ISSUE THREAD

**Issue by whipowill** [ZhengPeiRu21/mod-playerbots#60]
It's essential that this module work w/ this addon:

https://github.com/ike3/mangosbot-addon

The only thing I've found that might help me figure out what commands are available in it's current form is here:

https://github.com/ZhengPeiRu21/mod-playerbots/issues/5

**Comment by ZhengPeiRu21** [comment]
Because this is ported from same IKE3 source, the addon should only need minor adjustment. I think the largest difference is AzerothCore playerbot commands begin with ".playerbot" prefix - if addon is changed to account for this, much of it I think should work.

**Comment by whipowill** [comment]
I will play with it and see if I can fix it.

**Comment by whipowill** [comment]
I've started making progress on this, not totally working yet but it's a start:
https://github.com/whipowill/wow-addon-playerbots

**Comment by ZhengPeiRu21** [comment]
Thank you for starting on updating the addon! When it is ready to level of would like others to test, please let me know and I will include it in the README for the module.

**Comment by whipowill** [comment]
I repushed this addon as force to overwrite what I had before, I had started w/ the wrong code.  Now starting w/ the right code, I only had to make minimal changes.  I think this new version is ready to go.  

The issues I'm experiencing now is that the bot itself, server-side, just won't do what I am telling him to do -- but according to this addon, the commands are being sent correctly and are being received correctly, it's just the bot code server-side won't do it right.  Also the other random bots are retarded and don't look like they attack well -- lots of standing around acting weird.
